src/sdkClient/sdkClient.ts

```diff
@@ -39,6 +39,8 @@ export function sdkClientFactory(params: ISdkFactoryContext, isSharedClient?: bo
 
           // Release the API Key if it is the main client
           if (!isSharedClient) releaseApiKey(settings.core.authorizationKey);
+          
+          if (uniqueKeysTracker && uniqueKeysTracker.stopFilterCleaner) uniqueKeysTracker.stopFilterCleaner();
```

Comment:
Replace with `if (uniqueKeysTracker) uniqueKeysTracker.stop();` if doing the `stop` method required